### ISSUE TITLE
English English Dictionary

### ISSUE THREAD

**Issue by expacc007** [FooSoft/yomichan#2126]
I am a non native english speaker. I want to use ajatt principles to aid my learning. I want to english but I really like the fast yomichan to anki flashcard generation. So, is there any way to use English English Dictionary with yomichan?

**Comment by Ceynou** [comment]
I guess you could use yomichan with a EN-EN compatible dictionary and some tweaking but yomichan was made for japanese, so there's no english deconjugation or whatever.
you might want to use [vocabsieve](https://github.com/FreeLanguageTools/vocabsieve/)?

**Comment by expacc007** [comment]
Yeah, I am trying out vocabseive since I can't find a EN-EN compatible dictionary and don't know what tweaks to make.
